### DISCUSSION TITLE
Adafruit USB Host option available for all RP2040 boards but the Raspberry Pi Pico?

### DISCUSSION THREAD

**Discussion by rppicomidi** [earlephilhower/arduino-pico#1920] [Q&A] [answered]
Forgive me for this newbie question, but I have just started to use the Arduino IDE and was very excited to try the native RP2040
USB Host option that was recently committed but has not yet put in a tagged release. I am running the Arduino IDE version 2.2.1 from a Linux App Image installation on a PC running Ubuntu 22.04LTS. I installed arduino-pico board manager package release 3.6.2 and verified everything is working OK. Then I copied the latest `boards.txt` file recently committed by @hathach over the file found in `${HOME}/.arduino15/packages/rp2040/hardware/rp2040/3.6.2/`. Under the IDE Tools menu, the USB Stack: options for Adafruit TinyUSB Host appear for the Raspberry Pico W and every other board I tried, but not the Raspberry Pi Pico. Is there a right way to do what I am trying to do?

**Comment by earlephilhower** [comment, marked as answer]
There's a new wifi AP bug I want to look at (#1918) before doing a release, but it should only be a few days until it's out.

Until then I would suggest using the `git` method to clone this repo on your machine and use the local version instead of editing an older release.  It's documented [here](https://github.com/earlephilhower/arduino-pico/blob/master/docs/install.rst#installing-via-git).  Good luck!

**Reply by rppicomidi** [reply]
Thank you! All works as expected now. I had tried the instructions in the link you provided previously, and they did not work because I did not realize that when the instructions said `~/Arduino` it meant the sketch directory (my sketches are in `~/Documents/Arduino`). Now it
is working as expected.
